src/main/java/com/vmware/vipclient/i18n/base/instances/TranslationMessage.java

```diff
@@ -249,6 +253,38 @@ public boolean postString(final Locale locale, final String component,
 		return cs.getComponentTranslation();
 	}
 
+
+    /**
+     * get multiple components' translations from VIP server
+     *
+     * @param locales
+     *            locales to get the translations of them
+     * @param components
+     *            names of the components to get translation
+     * @return 
+                  a map contains all translations of the components of specified locales.
+		  Key is loale; value is also a map whose key is component and value is the messages belong to this component.
+     */
+    public Map<Locale, Map<String, Map<String, String>>> getStrings(final Set<Locale> locales,
+            final Set<String> components) {
+        logger.info("Start to execute TranslationMessage.getStrings of multiple components of multiple locales.");
```

Comment:
logger.info -> logger.debug

Reply:
This method is not used frequently, so it's acceptable. It's also good that we have message in log to identify problems.